### DISCUSSION TITLE
Welcome to tinycore-redpill Discussions!

### DISCUSSION THREAD

**Discussion by pocopico** [pocopico/tinycore-redpill#292] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by damienmillet** [comment]
a wiki can be nice too with `how to` section

- how to prepare key (into redpill) manually
- how to update dsm and loader manually 
- how to do it with web interface

**Comment by Snaiper4** [comment]
Good afternoon. I wish to use it as a motherboard https://ali.onl/2aMy . Questions: 1. Will the 12th generation video core work correctly? 2. Will the network interfaces presented on this board work correctly? 3. Which device emulation is better to use?

**Reply by damienmillet** [reply]
Hi, 
> 12th generation video core work 

no, driver aren't back ported to work. dsm kernel's 4.4 and 12th're > 5.x
Can u send the model of your motherboard, my Adblock deny me.

**Reply by Snaiper4** [reply]
https://www.aliexpress.com/item/1005005296599962.html noname motherboard

**Reply by damienmillet** [reply]
the I226-V is include in [igc](https://github.com/pocopico/rp-ext/tree/main/igc) and ll be added automatically ;) 
source : https://github.com/jim3ma/synology-igc

**Reply by Snaiper4** [reply]
thank you so much for the information! It is a pity that hardware transcoding will not work on Intel Celeron J6413(((

**Comment by Snaiper4** [comment]
hello friend! Be kind, as the loader is updated, to update and supplement the operating instructions.

**Comment by pocopico** [comment]
We could use some help with that, since you are enjoying the hard work of maintaining the loader, are you willing to maybe contribute on that ?

**Reply by Snaiper4** [reply]
maybe

**Comment by Snaiper4** [comment]
Question: Will hardware transcoding work on the asus z490 i and i3 10100 setup?

**Comment by s0rcier911** [comment]
I must be stupid, im running vmware, make a vm, started tinypill, open an ssh connection ... i have an ip adress assign by my router if i do ifconfig, but i always get a msg no route to host -> 
 'https://github.com/pocopico/tinycore-redpill.git/': Failed to connect to github.com port 443: No route to host

**Comment by FoldZero** [comment]
Setting Languge to C

How do I set the language back to English on TCRP App in Tiny?

**Comment by NickSouzaDasilva** [comment]
Bom dia
Estou com um R710 da Dell porem ao carregar o boot do tinycore redpill ele não identifica o drive de rede, não podendo eu acha-lo na rede para começar o processo de configuração.
Alguém tem alguma solução?
Obs: o mesmo pendrive funciona em outra maquina mais fraca, creio que deve estar faltando o driver de rede para esse modelo de servidor.
